Solutions/Cyborg Security HUNTER/Data/Solution_CyborgSecurity_HUNTER.json

```diff
@@ -0,0 +1,27 @@
+{
+    "Name": "Cyborg Security HUNTER",
+    "Author": "Mike Mitchell - mike@cyborgsecurity.com",
+    "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/cyborgsecurity-logo-75px.svg\" width=\"75px\" height=\"75px\">",
+    "Description": "The [Cyborg Security HUNTER](https://www.cyborgsecurity.com/) solution for Microsoft Sentinel helps analysts to configure the 'Open in Tool' button within the HUNTER platform, allowing the Sentinel hunt packages to be deployed in the Sentinel Platform",
```

Comment:
Please use Microsoft Sentinel instead of Sentinel and then repackage the solution